crates/ruff/src/rules/pandas_vet/rules/inplace_argument.rs

```diff
@@ -8,6 +8,29 @@ use crate::rules::pandas_vet::fixes::fix_inplace_argument;
 use crate::violation::AlwaysAutofixableViolation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for `inplace=True` in code using the `pandas` library.
+    ///
+    /// ## Why is this bad?
+    /// - `inplace=True` often does not provide a performance benefit. It is
+    /// likely to copy dataframes in the background.
```

Comment:
I think Markdown lists look a bit better if the wrapped text is indented as much as the first line of the list item.